vpd-tool/src/vpd_tool_main.cpp

```diff
@@ -0,0 +1,88 @@
+#include "constants.hpp"
+
+#include <CLI/CLI.hpp>
+
+#include <filesystem>
+#include <iostream>
+
+int main(int argc, char** argv)
+{
+    int l_retValue = 0;
+    try
+    {
+        CLI::App l_app{"VPD Command Line Tool"};
+
+        std::string l_vpdPath{};
+        std::string l_recordName{};
+        std::string l_keywordName{};
+        std::string l_filePath{};
+
+        l_app.footer(
+            "Read : DBus: "
+            "vpd-tool -r -O <Object Path> -R <Record Name> -K <Keyword Name>\n"
+            "Read : Hardware : "
+            "vpd-tool -r -H -O <EEPROM Path> -R <Record Name> -K <Keyword Name>\n");
+
+        auto l_objectOption = l_app.add_option(
+            "--object, -O", l_vpdPath, "Object Path or EEPROM File Path");
+        auto l_recordOption = l_app.add_option("--record, -R", l_recordName,
+                                               "Record Name");
+        auto l_keywordOption = l_app.add_option("--keyword, -K", l_keywordName,
+                                                "Keyword Name");
+
+        auto l_fileOption = l_app.add_option(
+            "--file", l_filePath,
+            "Absolute file path. To store readKeyword output or to fetch input value for writeKeyword.");
+
+        auto l_readFlag =
+            l_app.add_flag("--readKeyword, -r", "Read Keyword value")
+                ->needs(l_objectOption)
+                ->needs(l_recordOption)
+                ->needs(l_keywordOption);
+
+        auto l_hardwareOption =
+            l_app.add_flag("--Hardware, -H", "Read/Write from EEPROM. "
```

Comment:
-H alone is not enough to read/write from eeprom. 
so can we brief it clearly something like,
" Supportive flag to read from EEPROM / write to EEPROM."

Reply:
Lets discuss in the team, will update once finalised.